lib/book.rb

```diff
@@ -0,0 +1,21 @@
+class Book
+  attr_reader :author_first_name,
```

Comment:
Since we never call `.author_last_name` or `.author_first_name` as methods, we don't need them as attr_readers.